### ISSUE TITLE
liked songs

### ISSUE THREAD

**Issue by sowdust** [ldsz/plugin.audio.spotify#25]
Hi, is it me or is there no "liked songs" playlist? It would be great to have it among the other playlists!

**Comment by vguttmann** [comment]
Yeah, it's missing, but I think that's because it's not really a playlist. When I use other Spotify clients to list my playlists, it's missing as well, and you can't reorder songs in it like you can in regular playlists.

I think Spotify is doing something weird where it makes it look like a playlist, but it's not really a playlist, but it just compiles it from some other information source on the fly
